code/modules/mob/living/carbon/xenomorph/attack_alien.dm

```diff
@@ -58,6 +58,10 @@
 	playsound(loc, 'sound/weapons/alien_knockdown.ogg', 25, 1)
 
 	var/tackle_pain = X.xeno_caste.tackle_damage
+	if(X.frenzy_aura)
```

Comment:
this part is still frenzy

Reply:
don't tell me to remove code that should stay ahhhhhhh

Reply:
i put it on the line that needed to stay but you just removed everything in that block